Game/Pregnancy/Child.gd

```diff
@@ -79,6 +99,8 @@ func saveData():
 	return data
 
 func loadData(data):
+	name = SAVE.loadVar(data, "name", "")
```

Comment:
if the loaded name is empty you should generate a new random one
name = SAVE.loadVar...
if(name == ""):
    generateName()